src/points/PointsHistoryBottomSheet.tsx

```diff
@@ -0,0 +1,182 @@
+import React, { useMemo } from 'react'
+import { ActivityIndicator, StyleSheet, Text, View } from 'react-native'
+import SectionHead from 'src/components/SectionHead'
+import GorhomBottomSheet from '@gorhom/bottom-sheet'
+import { useDispatch, useSelector } from 'src/redux/hooks'
+import { pointsHistoryStatusSelector, pointsHistorySelector } from 'src/points/selectors'
+import { useTranslation } from 'react-i18next'
+import { typeScale } from 'src/styles/fonts'
+import Logger from 'src/utils/Logger'
+import BottomSheetBase from 'src/components/BottomSheetBase'
+import { Spacing } from 'src/styles/styles'
+import Attention from 'src/icons/Attention'
+import Colors from 'src/styles/colors'
+import Button, { BtnSizes, BtnTypes } from 'src/components/Button'
+import ValoraAnalytics from 'src/analytics/ValoraAnalytics'
+import { PointsEvents } from 'src/analytics/Events'
+import { getHistoryStarted } from 'src/points/slice'
+import { ClaimHistory } from 'src/points/types'
+import { groupFeedItemsInSections } from 'src/transactions/utils'
+import colors from 'src/styles/colors'
+import { BottomSheetSectionList } from '@gorhom/bottom-sheet'
+
+const TAG = 'Points/PointsHistoryBottomSheet'
+
+interface Props {
+  forwardedRef: React.RefObject<GorhomBottomSheet>
+}
+
+function PointsHistoryBottomSheet({ forwardedRef }: Props) {
+  const { t } = useTranslation()
+
+  const dispatch = useDispatch()
+
+  const pointsHistoryStatus = useSelector(pointsHistoryStatusSelector)
+  const pointsHistory = useSelector(pointsHistorySelector)
+
+  const onPressTryAgain = () => {
+    ValoraAnalytics.track(PointsEvents.points_screen_activity_try_again_press)
+    dispatch(
+      getHistoryStarted({
+        getNextPage: false,
+      })
+    )
+  }
+
+  const fetchMoreHistory = () => {
```

Comment:
could we prefix methods consistently with either `on` or `handle`? (my preference is `handle` for methods declared in the component and `on` for callbacks passed as props so that you can quickly understand where the logic is when reading the renders, although we've been super inconsistent about doing this)